Source/CarthageKitTests/ProjectSpec.swift

```diff
@@ -12,9 +12,113 @@ import Nimble
 import Quick
 import ReactiveSwift
 import Tentacle
+import Result
+import ReactiveTask
+import XCDBLD
 
 class ProjectSpec: QuickSpec {
 	override func spec() {
+		describe("createAndCheckVersionFiles") {
+			let directoryURL = Bundle(for: type(of: self)).url(forResource: "DependencyTest", withExtension: nil)!
+			let buildDirectoryURL = directoryURL.appendingPathComponent(CarthageBinariesFolderPath)
+			
+			func buildDependencyTest(platforms: Set<Platform> = [], cacheBuilds: Bool = true) -> Set<String> {
+				var builtSchemes: [String] = []
+				
+				let project = Project(directoryURL: directoryURL)
+				let result = project.buildCheckedOutDependenciesWithOptions(BuildOptions(configuration: "Debug", platforms: platforms, toolchain: "com.apple.dt.toolchain.Swift_2_3", cacheBuilds: cacheBuilds))
```

Comment:
Does this need to specify the toolchain? This test won't work for long if it requires Swift 2.3.